aergo/herapy/aergo.py

```diff
@@ -710,6 +711,23 @@ def get_tx_result(self, tx_hash):
 
         return tx_result
 
+    def wait_tx_result(self, tx_hash, timeout=30, tempo=0.2):
+        if self.__comm is None:
+            return None
+        if isinstance(tx_hash, str):
+            tx_hash = decode_tx_hash(tx_hash)
+        elif type(tx_hash) is th.TxHash:
+            tx_hash = bytes(tx_hash)
+
+        for _ in range(int(timeout/tempo)):
```

Comment:
How about trying "get_tx_result" even if "timeout" is smaller than "tempo"?

In this case, if 'int(timeout/tempo) == 0', then 'return None'.

Reply:
Sounds good, so you mean `for _ in range(int(timeout/tempo)+1):` ?

Reply:
yeah~seems like that